config/routes.rb

```diff
@@ -2,7 +2,9 @@
   root 'tattoos#index'
   devise_for :users, controllers: { registrations: :registrations }
 
-  resources :users, only: [:index, :show, :destroy]
+  resources :users, only: [:index, :show, :destroy] do
+    post 'make_admin'
```

Comment:
you should use a restful route for this action. Again just creating a custom route goes against Rails conventions. If you wanted for example, you could have a route:

```
resources :admin, only: [:create]
```

And that controller can update the user.